Lib/ufo2ft/__init__.py

```diff
@@ -35,6 +36,7 @@
 
 
 logger = logging.getLogger(__name__)
+timer = Timer(logging.getLogger("fontmake.timer"), level=logging.INFO)
```

Comment:
another thing, it's best we name this logger `"ufo2ft.timer"`, not `"fontmake.timer"`. ufo2ft is not only used inside fontmake.
It's ok that fontmake needs to become aware of this new logger in order to handle debug messages from it when the --timing flag is set. I have a local patch for fontmake which I will submit in a bit, that does that.